src/assets/icons/IconCamera.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M5.5 1H10.5L12.5 3H14C15.1 3 16 3.9 16 5V13C16 14.1 15.1 15 14 15H2C0.9 15 0 14.1 0 13V5C0 3.9 0.9 3 2 3H3.5L5.5 1ZM8 13C10.208 13 12 11.208 12 9C12 6.792 10.208 5 8 5C5.792 5 4 6.792 4 9C4 11.208 5.792 13 8 13ZM8 11C6.89543 11 6 10.1046 6 9C6 7.89543 6.89543 7 8 7C9.10457 7 10 7.89543 10 9C10 10.1046 9.10457 11 8 11Z" fill="#1D212A" fill-opacity="0.72"/>
+</svg>
```

Comment:
we already have a camera icon in CozyUI https://docs.cozy.io/cozy-ui/react/#!/Icon